modules/cluster-cloud-controller-config-osp.adoc

```diff
@@ -0,0 +1,271 @@
+
+| `ca-file`
+| Optional. The CA certificate bundle file for communication with the {rh-openstack} Identity service. If you use the HTTPS protocol with The Identity service URL, this option is required.
+
+| `cert-file`
+| Optional. The client certificate path to use for client TLS authentication.
+
+| `key-file`
+| Optional. The client private key path to use for client TLS authentication.
+
+| `username`
+| The Identity service user name. Leave this option unset if you are using Identity service application credentials.
+
+| `password`
+| The Identity service user password. Leave this option unset if you are using Identity service application credentials.
+
+| `region`
+| The Identity service region name.
+
+| `domain-id`
+| The Identity service user domain ID. Leave this option unset if you are using Identity service application credentials.
+
+| `domain-name`
+| The Identity service user domain name. This option is not required if you set `domain-id`.
+
+| `tenant-id`
+| The Identity service project ID. Leave this option unset if you are using Identity service application credentials.
+
+If you are using version 3 of the Identity API, which changed the identifier `tenant` to `project`, the value of `tenant-id` is automatically mapped to the project construct in the API. 
+
+| `tenant-name`
+| The Identity service project name. You can ignore this option if `tenant-id` is set.
+
+| `tenant-domain-id`
+| The Identity service project domain ID.
+
+| `tenant-domain-name`
+| The Identity service project domain name.
+
+| `user-domain-id`
+| The Identity service user domain ID.
+
+| `user-domain-name`
+| The Identity service user domain name.
+
+| `trust-id`
+| The Identity service trust ID. A trust represents the authorization of a user, or trustor, to delegate roles to another user, or trustee. Optionally, a trust authorizes the trustee to impersonate the trustor. You can find available trusts by querying the `/v3/OS-TRUST/trusts` endpoint of the Identity service API.
+
+| `trustee-id`
+| The Identity service trustee user ID.
+
+| `trustee-password`
+| The Identity service trustee user password.
+
+| `use-clouds`
+a| Set this option to `true` to fetch authorization credentials from a `clouds.yaml` file. Options set in this section are prioritized over values read from the `clouds.yaml` file. 
+
+CCM searches for the file in the following places:
+
+. The value of the `clouds-file` option.
+. A file path stored in the environment variable `OS_CLIENT_CONFIG_FILE`.
+. The directory `pkg/openstack`.
+. The directory `~/.config/openstack`.
+. The directory `/etc/openstack`.
+
+| `clouds-file`
+| The file path of a clouds.yaml file. It is used if the `use-clouds` option is set to `true`.
+
+| `cloud`
+| The named cloud in the `clouds.yaml` file that you want to use. It is used if the `use-clouds` option is set to `true`.
+
+| `application-credential-id`
```

Comment:
@imatza-rh these application-credentials options are being tested, correct?

Reply:
@MaysaMacedo hey, the `application-credential-id` and `application-credential-secret` options have been tested